src/Header.test.js

```diff
@@ -2,20 +2,36 @@ import React from "react";
 import { shallow, mount } from "enzyme";
 import Header from "./Header";
 
+const mockIsLoggedIn = jest.fn();
+
+beforeEach(() => {
+  mockIsLoggedIn.mockReset();
+});
+
 it("renders Header without crashing", () => {
-  mount(<Header />);
+  mockIsLoggedIn.mockReturnValueOnce(false);
+  mount(<Header isLoggedIn={() => mockIsLoggedIn} />);
 });
 
 it("renders correctly (snapshot testing)", () => {
-  const wrapper = mount(<Header />);
+  mockIsLoggedIn.mockReturnValueOnce(false);
+  const wrapper = mount(<Header isLoggedIn={mockIsLoggedIn} />);
+  expect(wrapper).toMatchSnapshot();
+
+  wrapper.unmount();
+});
+
+it("renders correctly logout (snapshot testing)", () => {
```

Comment:
should you rename the previous test to 'login'.

Reply:
Moreover, I don't think this is being tested right. Don't both the above tests generate the same snapshot file with something like <Header isLoggedIn=[Function]/>?

I don't know how to test this but this doesn't look right. Check the snapshot files.